plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/ElasticsearchServer.java

```diff
@@ -68,8 +73,21 @@ public void stop()
         deleteRecursively(configurationPath, ALLOW_INSECURE);
     }
 
+    @Override
+    public void close()
+            throws IOException
+    {
+        stop();
```

Comment:
do we still need public `stop()` method? (separate commit to make `ElasticsearchServer` closeable?)

Reply:
Got rid of `stop()` and extracted the change to a separate commit.